cached_resource.gemspec

```diff
@@ -17,8 +17,8 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = ">= 3.0"
 
-  s.add_runtime_dependency "activeresource", ">= 4.0"
-  s.add_runtime_dependency "activesupport", ">= 4.0"
+  s.add_runtime_dependency "activeresource", ">= 6.1"
+  s.add_runtime_dependency "msgpack", "~> 1.7", ">= 1.7.3"
```

Comment:
It looks like we still require active_support, for example in [lib/cached_resource.rb](https://github.com/mhgbrown/cached_resource/pull/72/files#diff-0a357eb04e75ecb6634b4d036a0177c1efca3a73b2d2130e64dfcce00509ead0) below. Is it Ok to remove it from the runtime dependencies?